tests/test_streaming_download_manager.py

```diff
@@ -298,7 +298,7 @@ def test_xlistdir(input_path, expected_paths, tmp_path, mock_fsspec):
 
 
 @pytest.mark.integration
-def test_xlistdir_private(hf_private_dataset_repo_zipped_txt_data, hf_token):
+def test_xlistdir_private(hf_private_dataset_repo_zipped_txt_data, hf_token, ci_hf_hub_url):
```

Comment:
Maybe the ci_hf_hub_url fixture can be included inside the hf_private_dataset_repo_zipped_txt_data one ? It already patches `datasets.config.HUB_DATASETS_URL` and `datasets.config.HF_ENDPOINT`

Reply:
I have included the ci_hf_hub_url fixture in the fixtures:
- hf_private_dataset_repo_txt_data
- hf_private_dataset_repo_zipped_txt_data
- hf_private_dataset_repo_zipped_img_data

Reply:
I have finally removed this fixture.